Makefile

```diff
@@ -33,145 +28,144 @@ datadir:
 
 # regression
 download-abalone: datadir
-	./download.sh http://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/regression/abalone_scale
```

Comment:
This is needed because wget is not available on OS X.

Reply:
I thought macOS is not supported because it is untested by CI.

Restored in 8f126082b5b5a766f66be68f76904363b5d0449d.